corehq/apps/cleanup/management/commands/delete_doc_conflicts.py

```diff
@@ -1,8 +1,22 @@
 from optparse import make_option
+from couchdbkit.exceptions import BulkSaveError
 from dimagi.utils.couch.database import iter_docs, get_db
 from django.core.management.base import BaseCommand, LabelCommand
 
 
+def bulk_delete(db, docs):
+    if not docs:
+        return
+
+    print("Deleting {} doc revisions".format(len(docs)))
+    try:
+        db.bulk_delete(docs)
+    except BulkSaveError:
+        print("BulkSaveError")
```

Comment:
also, I think BulkSaveError gives you access to a list of the docs which errored